thoth/common/openshift.py

```diff
@@ -1315,7 +1315,6 @@ def schedule_adviser(
         kebechet_metadata: Optional[Dict[str, Any]] = None,
         justification: Optional[List[Dict[str, Any]]] = None,
         stack_info: Optional[List[Dict[str, Any]]] = None,
-        package_update_info: Optional[Dict[str, str]] = None,
```

Comment:
what component was using this parameter?

Reply:
https://github.com/thoth-station/user-api/pull/1456

Was never actually pulled in

Reply:
Thanks, then I think we are good here :)